CHANGELOG.md

```diff
@@ -149,6 +149,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 - Fix circular dependency in Settings initialization ([10194](https://github.com/opensearch-project/OpenSearch/pull/10194))
 - Fix registration and initialization of multiple extensions ([10256](https://github.com/opensearch-project/OpenSearch/pull/10256))
 - Fix Segment Replication ShardLockObtainFailedException bug during index corruption ([10370](https://github.com/opensearch-project/OpenSearch/pull/10370))
+- Fix profile rewrite time issue in the concurrent path ([10352](https://github.com/opensearch-project/OpenSearch/pull/10352))
```

Comment:
Nit: I think the Changelog should be more descriptive than "fixed x issue"

Reply:
Hey @jed326, thanks for the review. Do you have any suggestions for this?

Reply:
I think a shortened version of the description you already added to the PR description above is good. Maybe something like:

"Fix timer race condition in profile rewrite and create weight for concurrent segment search"

Reply:
The change log has updated now. Thank you.